Library/Homebrew/cmd/readall.rb

```diff
@@ -34,12 +34,9 @@ def readall
     if ARGV.named.empty?
       formulae = Formula.full_names
     else
-      user, repo = tap_args
-      user.downcase!
-      repo.downcase!
-      tap = HOMEBREW_LIBRARY/"Taps/#{user}/homebrew-#{repo}"
-      raise "#{tap} does not exist!" unless tap.directory?
-      tap.find_formula { |f| formulae << f }
+      tap = Tap.new *tap_args
```

Comment:
Can we use parens here, this will raise a warning (I occasionally run with warnings enabled and fix the low-hanging fruit).